src/DevTools/ErrorPage.php

```diff
@@ -250,16 +261,42 @@ private function render_title() {
 	 * @return string KSES-sanitized message.
 	 */
 	private function render_message() {
-		return wp_kses_post( $this->message );
+		return '<p>' . wp_kses_post( $this->message ) . '</p>';
 	}
 
 	/**
-	 * Render the source of the exception file.
+	 * Render support link.
+	 *
+	 * @return string
+	 */
+	private function render_support_link() {
+		return '<p>' . wp_kses(
+			sprintf(
+				/* translators: %s is the AMP support forum URL. */
+				__( 'If you get stuck, you may want to share any details in a new topic on the plugin\'s <a href="%s" target="_blank" rel="noreferrer noopener">support forum</a>.', 'amp' ),
+				esc_url( __( 'https://wordpress.org/support/plugin/amp/', 'amp' ) )
```

Comment:
Do we actually want this link to be translatable? It could potentially allow an attacker to send the user to a "fishing support forum" where they post their logs to a third-party... Yes, it is far-fetched, I know.

Reply:
The URL was made translatable in https://github.com/ampproject/amp-wp/pull/6320 as well. There are these instances of translatable URLs:

```
includes/options/class-amp-options-manager.php
470:						esc_url( __( 'https://wordpress.org/support/article/why-should-i-use-https/', 'amp' ) )

assets/src/components/error-screen/index.js
49:						__( 'https://wordpress.org/support/plugin/amp/', 'amp' ),

assets/src/settings-page/analytics.js
26:	__( 'https://wordpress.org/plugins/google-site-kit/', 'amp' ),
27:	__( 'https://developers.google.com/analytics/devguides/collection/amp-analytics/', 'amp' ),
213:							__( 'https://amp.dev/documentation/components/amp-analytics/', 'amp' ),
215:							__( 'https://amp-wp.org/documentation/getting-started/analytics/', 'amp' ),
220:							__( 'https://amp.dev/documentation/guides-and-tutorials/optimize-and-measure/configure-analytics/analytics-vendors/', 'amp' ),

assets/src/settings-page/paired-url-structure.js
337:						__( 'https://amp-wp.org/?p=10004', 'amp' ),

src/Admin/SiteHealth.php
248:				esc_url( __( 'https://www.php.net/manual/en/book.curl.php', 'amp' ) ),

src/DevTools/ErrorPage.php
277:				esc_url( __( 'https://wordpress.org/support/plugin/amp/', 'amp' ) )

src/LoadingError.php
47:							esc_url( __( 'https://wordpress.org/support/plugin/amp/', 'amp' ) )
```

Granted, we aren't going to have alternative support forums so I'm fine with removing the translation in both places. 

cc @delawski